base/macros.hpp

```diff
@@ -163,6 +163,9 @@ inline void noreturn() { std::exit(0); }
 #  error "What compiler is this?"
 #endif
 
+// We assume that the processor is at least a Nehalem.
+#define PRINCIPIA_USE_SSE2_INTRINSICS 1
```

Comment:
Define this to !_DEBUG, so that we check that both compile.

Reply:
Done.